packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/object-method-with-super/expected.js

```diff
@@ -0,0 +1,11 @@
+class Foo extends class {} {
+  method() {
+    var _superprop_callMethod = (..._args) => super.method(..._args);
```

Comment:
This looks okay to me. Nice job, you use the same reference to the `super.method` call (instead of two separate) :+1: